docs/index.md

```diff
@@ -1,3 +1,41 @@
-# Open Podcast API specification hub
+# Open Podcast API
 
-This site contains the specifications and documentation for the Open Podcast API.
+Welcome! The Open Podcast API is an initiative aiming to provide a feature-complete synchronisation API specification for podcast (web) apps and user-focussed servers.
+
+Our goal indirectly serving end-users: providing an easy way to sync subscriptions, listening progress, favourites, queu(s) and more between different apps and online services, and the ability to switch providers without hassle. Our goal for developers: providing reliable, open specifications which are decentralisation-ready and are easy to implement. To reach these goals we're deveveloping these specs in a collaborative way and sharing them in an [OpenAPI](https://spec.openapis.org/oas/latest) format.
+
+## Supported feature set & 'alpha' status
+
+Currently the specifications to synchronise [subscriptions](/docs/subscriptions) has been made available.
```

Comment:
For some reason, `[subscriptions](/docs/subscriptions)` gets parsed as a link to "#/docs/subscriptions" and not "/docs/subscriptions". Docs to this are here: https://myst-parser.readthedocs.io/en/latest/syntax/cross-referencing.html#default-destination-resolution

Reply:
```suggestion
Currently the specifications to synchronise [subscriptions](subscriptions/index.md) has been made available.
```

Reply:
No, I couldn't get it to work this way - then it interprets `subscriptions` as a section on the current page. `(subscriptions/index.md)` seems to be the best we can do. Sphinx automatically inserts a link to the generated .html file of the Subscriptions page then.